site/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Redirecting</title>
+    <meta http-equiv="refresh" content="0; url=./docs" />
+  </head>
+  <body>
+    Redirecting to <a href="./docs">Blog/</a>...
```

Comment:
@deemp I think this might be not actual?

Reply:
@fizruk, see the gh-pages structure https://github.com/deemp/normalizer/tree/gh-pages. I need this `index.html` because I create a non-root `docs` directory.

Reply:
Then why is it redirecting to a blog?

Reply:
Because I copy-pasted it from rzk and didn't notice. Fixed in 94eb0baa9483f9995843397be664fdcfbc6a92b2